daemons/execd/cts-exec-helper.c

```diff
@@ -527,7 +527,7 @@ main(int argc, char **argv)
     gboolean use_tls = FALSE;
     crm_trigger_t *trig;
 
-    crm_log_cli_init("cts-exec-helper");
+    pcmk__cli_init_logging("cts-exec-helper", 0);
```

Comment:
This commit looks great, but change the commit message like "API: libcrmcommon: ...". With the new changelog section, that will look like:

    - Public API changes since 2.0.5
      + libcrmcommon: ...
      + ...